docs/source/software_process.rst

```diff
@@ -0,0 +1,64 @@
+..  _software_process:
+
+================
+Software Process
+================
+
+----------------------
+Continuous Integration
+----------------------
+
+PyNWB is tested against Ubuntu, macOS and Windows operating systems.
+The project has both unit and integration tests.
+
+* Circleci runs all PyNWB tests against Ubuntu
+* Appveyor runs all PyNWB tests against Windows
+* Travis runs all PyNWB tests against macOS
+
+Tests are ran everytime there is a new branch. That way the contributor knows
+that they are not breaking core functionality of PyNWB on all operating systems.
```

Comment:
That way, as a contributor you know if you introduced regressions or coding style inconsistencies.